build.sbt

```diff
@@ -69,12 +69,6 @@ ThisBuild / githubWorkflowBuildMatrixExclusions ++=
     ) // 3.0.0-M1 doesn't work on JDK 14+
   }
 
-// exclude DottyJS for now
```

Comment:
@djspiewak this exclusion was actually redundant

Reply:
I really really wanted excluded.